### ISSUE TITLE
How to document general R objects

### ISSUE THREAD

**Issue by berndbischl** [r-lib/roxygen2#151]
Hi,

in some of my packages I "precreate" a bunch R objects and document their meaning on a single help page (via rdname). 

But roxygen2 does now create a usage and even a format section on that page, which I do not want.

So i had to put 
@usage none
@format none
in front of every object definition. As copy/paste is always an indication that something is wrong, I am now asking for help here. 

I tried to figure out whether there is a specific docType for this, but only found "package" and "data". Maybe "robject" would be nice to add here?

Basically I do not want "anything more", I just want a simple way to suppress the above mentioned sections. Although I would guess that documenting R objects is a case general enough to warrant something like a docType for it?

Best,

Bernd
